src/System.Runtime.Extensions/tests/System/EnvironmentTests.cs

```diff
@@ -175,10 +175,18 @@ public void Version_MatchesFixedVersion()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap)] // Throws InvalidOperationException in Uap as NtQuerySystemInformation Pinvoke is not available
         public void WorkingSet_Valid()
         {
             Assert.True(Environment.WorkingSet > 0, "Expected positive WorkingSet value");
         }
+        
+        [Fact]
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.Uap)] 
+        public void WorkingSet_Valid_Uap()
+        {
+            Assert.Throws<InvalidOperationException>(() => Environment.WorkingSet);
```

Comment:
The way I removed the usage in Uap, Environment.WorkingSet tries to get process information on local machine from process counter (the way we were doing for remote machines) instead of throwing PNSE and then it throws `System.InvalidOperationException : Process performance counter is disabled, so the requested operation cannot be performed` when it isn't successful.